### DISCUSSION TITLE
Missing helpers for `justify-content: safe|unsafe center`

### DISCUSSION THREAD

**Discussion by silverwind** [tailwindlabs/tailwindcss#11879] [Ideas]
The [`justify-content` helpers](https://tailwindcss.com/docs/justify-content) are missing for the `safe` and `unsafe` keywords. See [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content#safe):

```css
justify-content: safe center;
justify-content: unsafe center;
```

Supported in Firefox 63+, Chrome 115+. `safe center` is very useful to avoid flex children from overflowing their parent flexbox.

**Comment by victormarcel09** [comment]
flex-start: Items are packed towards the start of the main axis.
flex-end: Items are packed towards the end of the main axis.
center: Items are centered along the main axis.
[PaybyPlateMa Login](https://www.paybyplatema.me/)

**Comment by silverwind** [comment]
`safe` and `unsafe` are also available to [align-items](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items#safe). I think tailwind could implement these helpers:

```
justify-safe-center
justify-unsafe-center
items-safe-center
items-unsafe-center
```

Theoretically they can be used for all alignment values, but I think they likely only make sense together with `center`.
